### PR TITLE
Update strings.js

### PR DESCRIPTION
**Pull request by despoinakaz**

### CODE REVIEW

**Comment by zaggino** [comment]
@telis93 can you review please?

File: src/nls/el/strings.js [hunk 1]
```diff
@@ -120,11 +132,15 @@ define({
 
     "OPEN_FILE"                         : "Άνοιγμα Αρχείου",
     "SAVE_FILE_AS"                      : "Αποθήκευση Αρχείου",
-    "CHOOSE_FOLDER"                     : "Επίλεξε ένα φάκελο",
+    "CHOOSE_FOLDER"                     : "Επέλεξε ένα φάκελο",
```
Comment (teldosas) [inline src/nls/el/strings.js]: That's a tough one but it is "Επίλεξε". [See here](https://www.translatum.gr/forum/index.php?topic=173354.0) :)

File: src/nls/el/strings.js [hunk 2]
```diff
@@ -271,17 +295,23 @@ define({
     "CMD_CSS_QUICK_EDIT_NEW_RULE"         : "Νέος Κανόνας",
     "CMD_NEXT_DOC"                        : "Επόμενο Έγγραφο",
     "CMD_PREV_DOC"                        : "Προηγούμενο Έγγραφο",
+    "CMD_NEXT_DOC_LIST_ORDER"             : "Επόμενο έγγραφο στην λίστα",
+    "CMD_PREV_DOC_LIST_ORDER"             : "Προηγούμενο έγγραφο στην λίστ",
```
Comment (teldosas) [inline src/nls/el/strings.js]: You missed an "α" here :)

File: src/nls/el/strings.js [hunk 3]
```diff
@@ -407,21 +440,41 @@ define({
     "DEBUG_MENU"                                : "Debug",
     "CMD_SHOW_DEV_TOOLS"                        : "Προβολή Developer Tools",
     "CMD_REFRESH_WINDOW"                        : "Επαναφόρτωση {APP_NAME}",
+    "CMD_RELOAD_WITHOUT_USER_EXTS"              : "Επαναφόρτωση χωρίς τις επεκτάσεις",
     "CMD_NEW_BRACKETS_WINDOW"                   : "Νέο Παράθυρο {APP_NAME}",
+    "CMD_LAUNCH_SCRIPT_MAC"                     : "Εγκάτασταση συντόμευσης για την γραμμή εντολών",
```
Comment (teldosas) [inline src/nls/el/strings.js]: Here it should be "Εγκατάσταση"

**Review by teldosas — CHANGES_REQUESTED** [review]
Ok @zaggino I went through the translation, I only found some small typos. Excellent work @despoinakaz :+1:

**Comment by telis93** [comment]
BTW @teldosas is my new account :wink:

**Comment by despoinakaz** [comment]
Thanks a lot @teldosas! I have made the changes.

**Review by teldosas — APPROVED** [review]

**Comment by petetnt** [comment]
Thanks!

**Pull request merged**